package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "blog",
+  "name": 'blog',
```

Comment:
you don't need to change this file, please revert the changes and add it to exclusions in Rubocop